src/components/Buttons/ButtonsTrack/ButtonsTrack.jsx

```diff
@@ -0,0 +1,11 @@
+import { Button } from '../Button/Button';
+import { BUTTONS_NAMES, BUTTONS_TYPES } from '../../../shared/constants';
+
+export function ButtonsTrack() {
+  return (
+    <>
+      <Button title={BUTTONS_NAMES.edit} stylingType={BUTTONS_TYPES.typeEdit} />
+      <Button title={BUTTONS_NAMES.delete} stylingType={BUTTONS_TYPES.typeDelete} />
```

Comment:
I'd recommend to create `ButtonsGroup` component instead and pass in it array with buttons descriptions as `items` params